http/oidc/pom.xml

```diff
@@ -148,6 +152,11 @@
             <artifactId>mockwebserver</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>${version.org.apache.httpcomponents.httpcore}</version>
```

Comment:
We don't need to specify the version here.

Reply:
Just a minor thing, but would be good to add this dependency underneath the `org.apache.httpclient` dependency.